dashboard/app/helpers/levels_helper.rb

```diff
@@ -45,6 +45,10 @@ def set_videos_and_blocks_and_callouts_and_instructions
       @start_blocks = initial_blocks(current_user, @level) || @start_blocks || @level.start_blocks
     end
 
+    if current_user && params[:load_previous]
+      @start_blocks = current_user.last_attempt(@level).try(:level_source).try(:data)
```

Comment:
I think when we actually ship this feature, the "loadprev/clean" button will want to be client side, so we'll have to pass both "start blocks" and "the user's previous solution" separately anyway.

(I was trying to do that here but I don't understand blockly well enough yet, working on that next)